homeassistant/components/ezviz/config_flow.py

```diff
@@ -160,9 +158,20 @@ async def async_step_user_camera(self, user_input=None):
             step_id="user_camera", data_schema=camera_schema, errors=errors or {}
         )
 
-    async def async_step_discovery(self, discovery_info=None):
-        """Handle a flow for discovered camera without rtsp entry."""
+    async def async_step_discovery(self, discovery_info):
+        """Handle a flow for discovered camera without rtsp config entry."""
         errors = {}
+        # This step is called when user goes to integrations,
+        # to complete config for discovered entry.
+        if discovery_info is not None:
```

Comment:
The discovery step should always be called with discovery info. So we don't need this check.

Reply:
Done.